### PR TITLE
Migrate pull off pytorch-linux-xenial-py3.7-gcc7

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: .github/workflows/pull.yml [hunk 1]
```diff
@@ -46,25 +46,20 @@ jobs:
...
+    name: linux-focal-py3.7-gcc7-no-ops
     uses: ./.github/workflows/_linux-build.yml
     with:
-      build-environment: linux-xenial-py3.7-gcc7
-      docker-image-name: pytorch-linux-xenial-py3.7-gcc7
+      build-environment: linux-focal-py3.7-gcc7-no-ops
+      docker-image-name: pytorch-linux-focal-py3.7-gcc7
 
-  linux-xenial-py3_7-gcc7-test:
+  # TODO: remove after full migration off Xenial
```
Comment (janeyx99) [inline .github/workflows/pull.yml:56]: when will that be?

**Review by janeyx99 — APPROVED** [review]

**Review by janeyx99 — COMMENTED** [review]
have we confirmed that there are no other jobs dependent on this docker image?

**Comment by kit1980** [comment]
> have we confirmed that there are no other jobs dependent on this docker image?

I checked that, should be ok.

**Pull request closed without merging**